package.json

```diff
@@ -3,9 +3,15 @@
   "version": "2.0.2",
   "description": "Sanitize your express payload to prevent MongoDB operator injection.",
   "main": "index.js",
+  "types": "index.d.ts",
+  "files": [
```

Comment:
the published package will only include `index.js` and `index.d.ts`.
So, we do not need to includes `index.test-d.ts` and `test.js`.